README.md

```diff
@@ -1,12 +1,38 @@
 # Probot: CODEOWNERS labeller
 
-> a GitHub App built with [Probot](https://github.com/probot/probot) that adds labels in Pull Requests based on contents of the CODEOWNERS.
+> a GitHub App built with [Probot](https://github.com/probot/probot) that adds labels in Pull Requests based on contents of the `codeowners-labeller.yml` file.
 
 ## Usage
 
 1. **[Install the app](https://github.com/apps/elastic-codeowners-labeller)**.
-2. Create a `CODEOWNERS` file in your repository, under the `.github` directory.
-3. Wait for new Pull Requests to be opened or synchronised.
+1. Create a `codeowners-labeller.yml` file in your repository, under the `.github` directory.
+1. Use your own labels as the keys in the YAML file.
+1. Use an array for paths to be added to a label. Pathname Format: glob
+1. Use the empty label (`- "":`) for exclusions. Paths here won't receive a label unless precedent labels apply.
+1. Wait for new Pull Requests to be opened or synchronised.
 
-## Important considerations
-The labels are based on Elastic criteria, so please update your [`labels.js`](./lib/labels.js) file in consequence.
+An example `codeowners-labeller.yml` file is shown here:
+
+```yaml
```

Comment:
I'd add some examples like `folder/**/*.md` to show an example of all supported glob paths